packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/edge/EdgeLabel.tsx

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import { GQLLabelStyle } from '../../graphql/subscription/labelFragment.types';
+import { EdgeLabelProps } from './EdgeLabeL.types';
+
+export const getFontWeight = (style: GQLLabelStyle | null): string => {
+  if (!!style?.bold) {
+    return 'bold';
+  }
+  return 'normal';
+};
+
+export const getFontSize = (style: GQLLabelStyle | null): number => {
+  return style?.fontSize ?? 14;
+};
+
+export const getFontStyle = (style: GQLLabelStyle | null): string => {
```

Comment:
It seems that function is not used.
Removes it in that case.
If this function is meant to be used do not export if it is not needed.